ci/scripts/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -eux
```

Comment:
I think the contents of this file just need to look something like, as the current code goes off to look for software from an s3 bucket which this pipeline doesn't do.

```

#!/bin/bash -eux

pushd dp-adot-collector
  cp Dockerfile.concourse ../build
popd
```